SignalServiceKit/src/Messages/OWSMessageManager.m

```diff
@@ -1884,6 +1884,15 @@ - (void)finalizeIncomingMessage:(TSIncomingMessage *)incomingMessage
     if (incomingMessage.isFriendRequest) {
         [thread removeOldIncomingFriendRequestMessagesIfNeededWithTransaction:transaction];
     }
+//    
+//    if (!CurrentAppContext().isMainApp) {
+//        dispatch_async(dispatch_get_main_queue(), ^{
+//            [self.typingIndicators didReceiveIncomingMessageInThread:masterThread
```

Comment:
If we're never going to be using this code again we can just delete it instead of commenting it out